docs/cycling/examples/cnr5.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: "atlassian.com/v1"
+kind: "CycleNodeRequest"
+metadata:
+  name: "example"
+  namespace: "kube-system"
+spec:
+  nodeGroupName: "az1-nodes.my-nodegroup.my-site"
```

Comment:
In this example what happens ? Is `nodeGroupName` the truth or `nodeGroupsList`? Should a CNR with both be invalid?

Reply:
Ah found it
> // buildNodeGroupNames builds a union of cloud provider node group names

This seems like reasonable backwards compatible behaviour 👍